File: lib/model/class.dart
class Class {
  final String companyId;
  final int createdAt;
  final String name;
  final String id;
  final String status;
  final int percentage;

  Class(
      {this.companyId,
      this.createdAt,
      this.id,
      this.name,
      this.percentage,
      this.status});

  factory Class.fromJson(Map<String, dynamic> json) {
    return Class(
        companyId: json["company_id"],
        createdAt: json["created_at"],
        id: json["id"],
        name: json["name"],
        percentage: json["summary"]["percentage"] ?? 0,
        status: json["status"]);
  }
}

File: app/src/main/java/com/guioliveiraapps/skore/model/Class.kt
package com.guioliveiraapps.skore.model

import com.google.gson.annotations.SerializedName

data class Class(
    @SerializedName("company_id") val companyId: String,
    @SerializedName("created_at") val createdAt: Long,
    val name: String,
    val id: String,
    val status: StatusEnum,
    val summary: Summary
)